rules/src/main/scala/fix/ForceOverrides.scala

```diff
@@ -75,7 +75,7 @@ final class ForceOverrides extends SemanticRule("fix.ForceOverrides") {
                 if (t.mods.map(_.toString).contains("override")) {
                   Patch.empty
                 } else {
-                  Patch.addLeft(t, "override ")
+                  Patch.replaceTree(t, t.copy(mods = t.mods :+ Mod.Override()).toString())
```

Comment:
This is significantly less efficient than `addLeft` but I think that reimplementing rules of methods modifiers priority is not a good idea.

Let's give it a shot and if it will turn out to be problematic we will think about other solutions, wdyt?